frontend/src/components/AdminSettings.vue

```diff
@@ -170,16 +185,61 @@
                   {{ t('admin.licenseInfo.seats.description.undercutSeats', [numberOfExceededSeats]) }}
                 </p>
               </div>
+
+              <div class="col-span-6 flex justify-end space-x-3">
+                <button type="button" class="flex-none inline-flex justify-center py-2 px-4 border border-transparent shadow-sm text-sm font-medium rounded-md text-white bg-primary hover:bg-primary-d1 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-primary disabled:opacity-50 disabled:hover:bg-primary disabled:cursor-not-allowed" @click="manageSubscription()">
+                  <ArrowTopRightOnSquareIcon class="-ml-1 mr-2 h-5 w-5" aria-hidden="true" />
+                  {{ t('admin.licenseInfo.getLicense') }}
+                </button>
+              </div>
             </div>
           </div>
         </div>
       </div>
 
-      <div v-if="!admin.hasLicense" class="flex justify-end items-center">
-        <button type="button" class="flex-none inline-flex justify-center py-2 px-4 border border-transparent shadow-sm text-sm font-medium rounded-md text-white bg-primary hover:bg-primary-d1 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-primary disabled:opacity-50 disabled:hover:bg-primary disabled:cursor-not-allowed" @click="manageSubscription()">
-          <ArrowTopRightOnSquareIcon class="-ml-1 mr-2 h-5 w-5" aria-hidden="true" />
-          {{ t('admin.licenseInfo.getLicense') }}
-        </button>
+      <div class="bg-white px-4 py-5 shadow sm:rounded-lg sm:p-6">
+        <div class="md:grid md:grid-cols-3 md:gap-6">
+          <div class="md:col-span-1">
+            <h3 class="text-lg font-medium leading-6 text-gray-900">
+              {{ t('admin.webOfTrust.title') }}
+            </h3>
+            <p class="mt-1 text-sm text-gray-500">
+              {{ t('admin.webOfTrust.description') }}
+            </p>
+          </div>
+          <div class="mt-5 md:mt-0 md:col-span-2">
```

Comment:
`div` can be a `form`, so the contained input fields and buttons are semantically grouped. Furthermore form validation might be easier this way (triggered by submit event instead of click event - so pressing enter works as well)